backend/wmg/pipeline/cube_pipeline.py

```diff
@@ -81,7 +81,7 @@ def main():
     # todo pass in validate_cubes as env arg
     try:
         snapshot_path, stats = load_data_and_create_cube("datasets", ".")
-        success_message = gen_wmg_pipeline_success_message(snapshot_path, stats)
+        success_message = gen_wmg_pipeline_success_message(snapshot_path, **stats)
```

Comment:
I think we also need to update the `gen_wmg_pipeline_success_message` method signature to make all the stats args into keyword parameters. Right now they're positional but dictionaries don't maintain order and ** passes as keyword args

Reply:
we can do `gen_wmg_pipeline_success_message(snapshot_path: str, *, dataset_count: int, cell_count: int, gene_count: int) -> dict:`

Reply:
nvm, apparently this isn't necessary after all--positional arguments can act as keyword arguments without the * notation